selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -9,15 +9,25 @@
 
 
 class Locators(object):
+
     ADD_PEOPLE_BUTTON = (By.XPATH, '//button[contains(.,"Add People")]')
-    ADD_PEOPLE_SEARCH_TXT = (By.ID, "emailHUID")
-    ADD_TO_COURSE_BUTTON = (By.ID, "add-user-btn-id")
-    ALERT_SUCCESS_ADD_PERSON = (By.XPATH, '//p[contains(.,"just added")]')
-    ALERT_SUCCESS_DELETE_PERSON = (By.XPATH, '//p[contains(.,"was just removed")]')
+    ADD_PEOPLE_SEARCH_TXT = (By.ID, 'emailHUID')
+    ADD_TO_COURSE_BUTTON = (By.ID, 'add-user-btn-id')
+    ALERT_SUCCESS_ALERT_BOX = (By.ID, 'alert-success')
+    ALERT_UNSUCCESSFUL_ADD_PERSON = (By.XPATH, '//div[contains(.,"could not '
+                                               'be added")]')
+    ALERT_SUCCESS_DELETE_PERSON = (By.XPATH, '//p[contains(.,"just removed")]')
     DELETE_USER_CONFIRM = (By.XPATH, '//button[contains(.,"Yes, Remove User")]')
+
+    MULTI_USER_PARTIAL_FAILURE_TEXT = (By.XPATH, '//div[contains(.,"person '
```

Comment:
This would only match if 1 person was not added -- seems like a flexible approach would be to collapse an equivalent to [the logic in the frontend](https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/blob/task/elliottyates/tlt-2119/add_people_allow_multiple/course_info/templates/course_info/partials/people.html#L153) into services on this PO called something like `people_added(n)` and `people_not_added(n)`, which will encapsulate checking for the right messages given the number of people.